validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -190,7 +190,8 @@ def unregister_peer(self, connection_id):
                                                      PeerStatus.TEMP)
             else:
                 LOGGER.debug("Attempt to unregister connection_id %s failed: "
-                             "connection_id was not registered")
+                             "connection_id was not registered",
```

Comment:
I'd put the connection id after the :, something like:

"Connection unregister failed as connection was not registered: %s"